xml/hardening.xml

```diff
@@ -4080,49 +4080,53 @@ drwxrwxrwt 18 root root 16384 Dec 23 22:20 /tmp</screen>
     <procedure>
      <step>
       <para>
-       Create a rules file
-       <filename>/etc/polkit-1/rules.d/01-restrict-removable-media.rules</filename>
-       similar to the following:
+       Create a user group whose users will be allowed to mount and eject
+       removable devices, for example <systemitem>mmedia_all</systemitem>:
       </para>
-      <screen>// Allow users in group 'mmedia_all' to mount/unmount all type of drives
-// Allow users in group 'mmedia_removable' to mount/umount USB storage drives
-// Allow users in group 'mmedia_optical' to mount/unmount Optical drives
+<screen>&prompt.sudo;groupadd mmedia_all</screen>
+     </step>
+     <step>
+      <para>
+       Add a specific user &exampleuser; to the new group:
+      </para>
+<screen>&prompt.sudo;usermod -a -G mmedia_all &exampleuser;</screen>
+     </step>
+     <step>
+      <para>
+       Create the <filename>/etc/polkit-1/rules.d/10-mount.rules</filename>
+       file with the following content:
+      </para>
+<screen>
+&prompt.user;cat /etc/polkit-1/rules.d/10-mount.rules
 polkit.addRule(function(action, subject) {
-  if (/^org\.freedesktop\.udisks2\.filesystem-.*mount.*$/.test(action.id) &amp;&amp;
-  action.lookup("drive.removable") == "true") {
-    if (subject.isInGroup("mmedia_all")) {
-      return polkit.Result.YES;
-    } else {
-      if (/.*optical.*/.test(action.lookup("drive.removable.media"))) {
-        if (subject.isInGroup("mmedia_optical"))
```

Comment:
I suggest flagging the group name with [replaceable]

Reply:
done